canonical-web-and-design/charmhub.io
# Change url on filter selection

## Done
- _Add a store layout, which can be re-used on other projects_
-  _Make filters persistent with the url_
-  _Add loading status on homepage_

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Change the filters and see they are persistent (if yu do hard refresh of the page, you still get the same results)
- Check it works fine on mobile as well

## Issue / Card
Fixes #679 

## Screenshots
[if relevant, include a screenshot]
